### ISSUE TITLE
Issue installing

### ISSUE THREAD

**Issue by sickboards** [bitpay/prestashop-v2#7]
Hi, we are running PS 1.7.5.2 and I can't seem to install the module,
I have tried it uploading 
prestashop-v2-master.zip
renaming it to 
bitpaycheckout.zip
rezipping it
it keeps saying: This file does not seem to be a valid module zip
I checked the config, it seems valid, even added the logo.gif but no luck.

**Comment by konors2k** [comment]
Hello, I'm having same issue with Prestashop 1.7.6.5 Module ( prestashop-v2-1.9.2020 ) cannot be installed through module installer of prestashop.

**Comment by sickboards** [comment]
Hi,

The module is only half working, the postbacks are not processed and the bitpay database table remains empty also I see the following error:
[Thu Dec 03 06:55:19.714034 2020] [proxy_fcgi:error] [pid 1475:tid 140489479071488] [client 2601:cb:8000:ea0:40e9:7abf:6e5d:2310:18620] AH01071: Got error 'PHP message: PHP Warning:  Creating default object from empty value in ../modules/bitpaycheckout/controllers/front/bitpayorder.php on line 121\n', referer: https://www.sickboards.nl/en/order
I guess that might be the root cause.

**Comment by info170** [comment]
Prestashop 1.7.6.8
Can't install module - got error "This file does not seem to be a valid module zip"

**Comment by M4ketech** [comment]
Yeah... Not working "This file does not seem to be a valid module .zip"
